// _@vuepress_core@1.5.2@@vuepress/core/.temp/internal/routes.js
/**
 * Generated by "@vuepress/internal-routes"
 */

import { injectComponentOption, ensureAsyncComponentsLoaded } from '@app/util'
import rootMixins from '@internal/root-mixins'
import GlobalLayout from "/Users/krialy/Desktop/mysite/blog/node_modules/_@vuepress_core@1.5.2@@vuepress/core/lib/client/components/GlobalLayout.vue"

injectComponentOption(GlobalLayout, 'mixins', rootMixins)
export const routes = [
  {
    name: "v-e3d2c21c",
    path: "/PAGE/I. 滑动窗口的最大值/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-e3d2c21c").then(next)
    },
  },
  {
    path: "/PAGE/I. 滑动窗口的最大值/index.html",
    redirect: "/PAGE/I. 滑动窗口的最大值/"
  },
  {
    path: "/LeetCode/2020-07-13-I.-%E6%BB%91%E5%8A%A8%E7%AA%97%E5%8F%A3%E7%9A%84%E6%9C%80%E5%A4%A7%E5%80%BC.html",
    redirect: "/PAGE/I. 滑动窗口的最大值/"
  },
  {
    name: "v-35f4ce68",
    path: "/PAGE/n个骰子的点数/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-35f4ce68").then(next)
    },
  },
  {
    path: "/PAGE/n个骰子的点数/index.html",
    redirect: "/PAGE/n个骰子的点数/"
  },
  {
    path: "/LeetCode/2020-07-17-n%E4%B8%AA%E9%AA%B0%E5%AD%90%E7%9A%84%E7%82%B9%E6%95%B0.html",
    redirect: "/PAGE/n个骰子的点数/"
  },
  {
    name: "v-36cdaa7e",
    path: "/PAGE/扑克牌中的顺子/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-36cdaa7e").then(next)
    },
  },
  {
    path: "/PAGE/扑克牌中的顺子/index.html",
    redirect: "/PAGE/扑克牌中的顺子/"
  },
  {
    path: "/LeetCode/2020-07-17-%E6%89%91%E5%85%8B%E7%89%8C%E4%B8%AD%E7%9A%84%E9%A1%BA%E5%AD%90.html",
    redirect: "/PAGE/扑克牌中的顺子/"
  },
  {
    name: "v-661ad32c",
    path: "/PAGE/I. 二叉树的深度/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-661ad32c").then(next)
    },
  },
  {
    path: "/PAGE/I. 二叉树的深度/index.html",
    redirect: "/PAGE/I. 二叉树的深度/"
  },
  {
    path: "/LeetCode/2020-07-26-I.-%E4%BA%8C%E5%8F%89%E6%A0%91%E7%9A%84%E6%B7%B1%E5%BA%A6.html",
    redirect: "/PAGE/I. 二叉树的深度/"
  },
  {
    name: "v-4986fb88",
    path: "/PAGE/求1+2+…+n/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-4986fb88").then(next)
    },
  },
  {
    path: "/PAGE/求1+2+…+n/index.html",
    redirect: "/PAGE/求1+2+…+n/"
  },
  {
    path: "/LeetCode/2020-07-26-%E6%B1%821+2+%E2%80%A6+n.html",
    redirect: "/PAGE/求1+2+…+n/"
  },
  {
    name: "v-0470c30a",
    path: "/PAGE/约瑟夫环/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-0470c30a").then(next)
    },
  },
  {
    path: "/PAGE/约瑟夫环/index.html",
    redirect: "/PAGE/约瑟夫环/"
  },
  {
    path: "/LeetCode/2020-07-28-%E7%BA%A6%E7%91%9F%E5%A4%AB%E7%8E%AF.html",
    redirect: "/PAGE/约瑟夫环/"
  },
  {
    name: "v-dfee1fac",
    path: "/PAGE/第五题/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-dfee1fac").then(next)
    },
  },
  {
    path: "/PAGE/第五题/index.html",
    redirect: "/PAGE/第五题/"
  },
  {
    path: "/LeetCode/2020-7-10.html",
    redirect: "/PAGE/第五题/"
  },
  {
    name: "v-21b1901c",
    path: "/PAGE/二进制中1的个数/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-21b1901c").then(next)
    },
  },
  {
    path: "/PAGE/二进制中1的个数/index.html",
    redirect: "/PAGE/二进制中1的个数/"
  },
  {
    path: "/LeetCode/2020-7-11-%E4%BA%8C%E8%BF%9B%E5%88%B6%E4%B8%AD1%E7%9A%84%E4%B8%AA%E6%95%B0.html",
    redirect: "/PAGE/二进制中1的个数/"
  },
  {
    name: "v-87ef0f38",
    path: "/PAGE/删除链表的节点/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-87ef0f38").then(next)
    },
  },
  {
    path: "/PAGE/删除链表的节点/index.html",
    redirect: "/PAGE/删除链表的节点/"
  },
  {
    path: "/LeetCode/2020-7-11-%E5%88%A0%E9%99%A4%E9%93%BE%E8%A1%A8%E7%9A%84%E8%8A%82%E7%82%B9.html",
    redirect: "/PAGE/删除链表的节点/"
  },
  {
    name: "v-454bcac6",
    path: "/PAGE/反转链表/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-454bcac6").then(next)
    },
  },
  {
    path: "/PAGE/反转链表/index.html",
    redirect: "/PAGE/反转链表/"
  },
  {
    path: "/LeetCode/2020-7-11-%E5%8F%8D%E8%BD%AC%E9%93%BE%E8%A1%A8.html",
    redirect: "/PAGE/反转链表/"
  },
  {
    name: "v-53e71f80",
    path: "/PAGE/合并两个排序的链表/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-53e71f80").then(next)
    },
  },
  {
    path: "/PAGE/合并两个排序的链表/index.html",
    redirect: "/PAGE/合并两个排序的链表/"
  },
  {
    path: "/LeetCode/2020-7-11-%E5%90%88%E5%B9%B6%E4%B8%A4%E4%B8%AA%E6%8E%92%E5%BA%8F%E7%9A%84%E9%93%BE%E8%A1%A8.html",
    redirect: "/PAGE/合并两个排序的链表/"
  },
  {
    name: "v-1429d8d2",
    path: "/PAGE/打印从1到最大的n位数/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-1429d8d2").then(next)
    },
  },
  {
    path: "/PAGE/打印从1到最大的n位数/index.html",
    redirect: "/PAGE/打印从1到最大的n位数/"
  },
  {
    path: "/LeetCode/2020-7-11-%E6%89%93%E5%8D%B0%E4%BB%8E1%E5%88%B0%E6%9C%80%E5%A4%A7%E7%9A%84n%E4%BD%8D%E6%95%B0.html",
    redirect: "/PAGE/打印从1到最大的n位数/"
  },
  {
    name: "v-4e50cc04",
    path: "/PAGE/斐波那契数列/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-4e50cc04").then(next)
    },
  },
  {
    path: "/PAGE/斐波那契数列/index.html",
    redirect: "/PAGE/斐波那契数列/"
  },
  {
    path: "/LeetCode/2020-7-11-%E6%96%90%E6%B3%A2%E9%82%A3%E5%A5%91%E6%95%B0%E5%88%97.html",
    redirect: "/PAGE/斐波那契数列/"
  },
  {
    name: "v-09c4483a",
    path: "/PAGE/旋转数组的最小数字/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-09c4483a").then(next)
    },
  },
  {
    path: "/PAGE/旋转数组的最小数字/index.html",
    redirect: "/PAGE/旋转数组的最小数字/"
  },
  {
    path: "/LeetCode/2020-7-11-%E6%97%8B%E8%BD%AC%E6%95%B0%E7%BB%84%E7%9A%84%E6%9C%80%E5%B0%8F%E6%95%B0%E5%AD%97.html",
    redirect: "/PAGE/旋转数组的最小数字/"
  },
  {
    name: "v-ac7a084e",
    path: "/PAGE/调整数组顺序使奇数位于偶数前面/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-ac7a084e").then(next)
    },
  },
  {
    path: "/PAGE/调整数组顺序使奇数位于偶数前面/index.html",
    redirect: "/PAGE/调整数组顺序使奇数位于偶数前面/"
  },
  {
    path: "/LeetCode/2020-7-11-%E8%B0%83%E6%95%B4%E6%95%B0%E7%BB%84%E9%A1%BA%E5%BA%8F%E4%BD%BF%E5%A5%87%E6%95%B0%E4%BD%8D%E4%BA%8E%E5%81%B6%E6%95%B0%E5%89%8D%E9%9D%A2.html",
    redirect: "/PAGE/调整数组顺序使奇数位于偶数前面/"
  },
  {
    name: "v-e622bd50",
    path: "/PAGE/链表中倒数第k个节点/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-e622bd50").then(next)
    },
  },
  {
    path: "/PAGE/链表中倒数第k个节点/index.html",
    redirect: "/PAGE/链表中倒数第k个节点/"
  },
  {
    path: "/LeetCode/2020-7-11-%E9%93%BE%E8%A1%A8%E4%B8%AD%E5%80%92%E6%95%B0%E7%AC%ACk%E4%B8%AA%E8%8A%82%E7%82%B9.html",
    redirect: "/PAGE/链表中倒数第k个节点/"
  },
  {
    name: "v-097457ce",
    path: "/PAGE/青蛙跳台阶问题/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-097457ce").then(next)
    },
  },
  {
    path: "/PAGE/青蛙跳台阶问题/index.html",
    redirect: "/PAGE/青蛙跳台阶问题/"
  },
  {
    path: "/LeetCode/2020-7-11-%E9%9D%92%E8%9B%99%E8%B7%B3%E5%8F%B0%E9%98%B6%E9%97%AE%E9%A2%98.html",
    redirect: "/PAGE/青蛙跳台阶问题/"
  },
  {
    name: "v-c12794c4",
    path: "/PAGE/两个链表的第一个公共节点/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-c12794c4").then(next)
    },
  },
  {
    path: "/PAGE/两个链表的第一个公共节点/index.html",
    redirect: "/PAGE/两个链表的第一个公共节点/"
  },
  {
    path: "/LeetCode/2020-7-12-%E4%B8%A4%E4%B8%AA%E9%93%BE%E8%A1%A8%E7%9A%84%E7%AC%AC%E4%B8%80%E4%B8%AA%E5%85%AC%E5%85%B1%E8%8A%82%E7%82%B9.html",
    redirect: "/PAGE/两个链表的第一个公共节点/"
  },
  {
    name: "v-046db744",
    path: "/PAGE/二叉树的镜像/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-046db744").then(next)
    },
  },
  {
    path: "/PAGE/二叉树的镜像/index.html",
    redirect: "/PAGE/二叉树的镜像/"
  },
  {
    path: "/LeetCode/2020-7-12-%E4%BA%8C%E5%8F%89%E6%A0%91%E7%9A%84%E9%95%9C%E5%83%8F.html",
    redirect: "/PAGE/二叉树的镜像/"
  },
  {
    name: "v-3e49954c",
    path: "/PAGE/从上到下打印二叉树 II/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-3e49954c").then(next)
    },
  },
  {
    path: "/PAGE/从上到下打印二叉树 II/index.html",
    redirect: "/PAGE/从上到下打印二叉树 II/"
  },
  {
    path: "/LeetCode/2020-7-12-%E4%BB%8E%E4%B8%8A%E5%88%B0%E4%B8%8B%E6%89%93%E5%8D%B0%E4%BA%8C%E5%8F%89%E6%A0%91-II.html",
    redirect: "/PAGE/从上到下打印二叉树 II/"
  },
  {
    name: "v-beafb7aa",
    path: "/PAGE/0～n-1中缺失的数字/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-beafb7aa").then(next)
    },
  },
  {
    path: "/PAGE/0～n-1中缺失的数字/index.html",
    redirect: "/PAGE/0～n-1中缺失的数字/"
  },
  {
    path: "/LeetCode/2020-7-13-0%EF%BD%9En-1%E4%B8%AD%E7%BC%BA%E5%A4%B1%E7%9A%84%E6%95%B0%E5%AD%97.html",
    redirect: "/PAGE/0～n-1中缺失的数字/"
  },
  {
    name: "v-7a794c28",
    path: "/PAGE/第一个只出现一次的字符/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-7a794c28").then(next)
    },
  },
  {
    path: "/PAGE/第一个只出现一次的字符/index.html",
    redirect: "/PAGE/第一个只出现一次的字符/"
  },
  {
    path: "/LeetCode/2020-7-12-%E7%AC%AC%E4%B8%80%E4%B8%AA%E5%8F%AA%E5%87%BA%E7%8E%B0%E4%B8%80%E6%AC%A1%E7%9A%84%E5%AD%97%E7%AC%A6.html",
    redirect: "/PAGE/第一个只出现一次的字符/"
  },
  {
    name: "v-7f2f0d1f",
    path: "/PAGE/和为s的两个数字/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-7f2f0d1f").then(next)
    },
  },
  {
    path: "/PAGE/和为s的两个数字/index.html",
    redirect: "/PAGE/和为s的两个数字/"
  },
  {
    path: "/LeetCode/2020-7-13-%E5%92%8C%E4%B8%BAs%E7%9A%84%E4%B8%A4%E4%B8%AA%E6%95%B0%E5%AD%97.html",
    redirect: "/PAGE/和为s的两个数字/"
  },
  {
    name: "v-40812629",
    path: "/PAGE/II. 左旋转字符串/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-40812629").then(next)
    },
  },
  {
    path: "/PAGE/II. 左旋转字符串/index.html",
    redirect: "/PAGE/II. 左旋转字符串/"
  },
  {
    path: "/LeetCode/2020-7-13-II.-%E5%B7%A6%E6%97%8B%E8%BD%AC%E5%AD%97%E7%AC%A6%E4%B8%B2.html",
    redirect: "/PAGE/II. 左旋转字符串/"
  },
  {
    name: "v-71ea67b5",
    path: "/PAGE/在排序数组中查找数字 I/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-71ea67b5").then(next)
    },
  },
  {
    path: "/PAGE/在排序数组中查找数字 I/index.html",
    redirect: "/PAGE/在排序数组中查找数字 I/"
  },
  {
    path: "/LeetCode/2020-7-13-%E5%9C%A8%E6%8E%92%E5%BA%8F%E6%95%B0%E7%BB%84%E4%B8%AD%E6%9F%A5%E6%89%BE%E6%95%B0%E5%AD%97-I.html",
    redirect: "/PAGE/在排序数组中查找数字 I/"
  },
  {
    name: "v-0aad27d6",
    path: "/PAGE/第一题/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-0aad27d6").then(next)
    },
  },
  {
    path: "/PAGE/第一题/index.html",
    redirect: "/PAGE/第一题/"
  },
  {
    path: "/LeetCode/2020-7-7.html",
    redirect: "/PAGE/第一题/"
  },
  {
    name: "v-87f4124e",
    path: "/PAGE/翻转单词顺序/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-87f4124e").then(next)
    },
  },
  {
    path: "/PAGE/翻转单词顺序/index.html",
    redirect: "/PAGE/翻转单词顺序/"
  },
  {
    path: "/LeetCode/2020-7-13-%E7%BF%BB%E8%BD%AC%E5%8D%95%E8%AF%8D%E9%A1%BA%E5%BA%8F.html",
    redirect: "/PAGE/翻转单词顺序/"
  },
  {
    name: "v-45b31136",
    path: "/PAGE/第二题/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-45b31136").then(next)
    },
  },
  {
    path: "/PAGE/第二题/index.html",
    redirect: "/PAGE/第二题/"
  },
  {
    path: "/LeetCode/2020-7-8.html",
    redirect: "/PAGE/第二题/"
  },
  {
    name: "v-66dcc99c",
    path: "/PAGE/第四题/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-66dcc99c").then(next)
    },
  },
  {
    path: "/PAGE/第四题/index.html",
    redirect: "/PAGE/第四题/"
  },
  {
    path: "/LeetCode/2020-7-9(2).html",
    redirect: "/PAGE/第四题/"
  },
  {
    name: "v-fe8e0ad4",
    path: "/PAGE/第三题/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-fe8e0ad4").then(next)
    },
  },
  {
    path: "/PAGE/第三题/index.html",
    redirect: "/PAGE/第三题/"
  },
  {
    path: "/LeetCode/2020-7-9.html",
    redirect: "/PAGE/第三题/"
  },
  {
    name: "v-4aa408a5",
    path: "/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-4aa408a5").then(next)
    },
  },
  {
    path: "/index.html",
    redirect: "/"
  },
  {
    name: "v-7e37de28",
    path: "/PAGE/JS原型与原型链/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-7e37de28").then(next)
    },
  },
  {
    path: "/PAGE/JS原型与原型链/index.html",
    redirect: "/PAGE/JS原型与原型链/"
  },
  {
    path: "/%E5%89%8D%E7%AB%AF/javascript/2020-07-17-JS%E5%8E%9F%E5%9E%8B%E4%B8%8E%E5%8E%9F%E5%9E%8B%E9%93%BE.html",
    redirect: "/PAGE/JS原型与原型链/"
  },
  {
    name: "v-7dabfa02",
    path: "/PAGE/实现基础的Vuejs/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-7dabfa02").then(next)
    },
  },
  {
    path: "/PAGE/实现基础的Vuejs/index.html",
    redirect: "/PAGE/实现基础的Vuejs/"
  },
  {
    path: "/Vuejs/2020-08-08-%E5%AE%9E%E7%8E%B0%E5%9F%BA%E7%A1%80%E7%9A%84Vuejs.html",
    redirect: "/PAGE/实现基础的Vuejs/"
  },
  {
    name: "v-55b0911b",
    path: "/PAGE/函数节流与防抖/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-55b0911b").then(next)
    },
  },
  {
    path: "/PAGE/函数节流与防抖/index.html",
    redirect: "/PAGE/函数节流与防抖/"
  },
  {
    path: "/%E5%89%8D%E7%AB%AF/javascript/2020-07-17-%E5%87%BD%E6%95%B0%E8%8A%82%E6%B5%81%E4%B8%8E%E9%98%B2%E6%8A%96.html",
    redirect: "/PAGE/函数节流与防抖/"
  },
  {
    name: "v-28b6d4e6",
    path: "/PAGE/实现私有方法/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-28b6d4e6").then(next)
    },
  },
  {
    path: "/PAGE/实现私有方法/index.html",
    redirect: "/PAGE/实现私有方法/"
  },
  {
    path: "/%E5%89%8D%E7%AB%AF/javascript/2020-07-17-%E5%AE%9E%E7%8E%B0%E7%A7%81%E6%9C%89%E6%96%B9%E6%B3%95.html",
    redirect: "/PAGE/实现私有方法/"
  },
  {
    name: "v-14d8253b",
    path: "/PAGE/深拷贝与函数只执行一次/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-14d8253b").then(next)
    },
  },
  {
    path: "/PAGE/深拷贝与函数只执行一次/index.html",
    redirect: "/PAGE/深拷贝与函数只执行一次/"
  },
  {
    path: "/%E5%89%8D%E7%AB%AF/javascript/2020-07-17-%E6%B7%B1%E6%8B%B7%E8%B4%9D%E4%B8%8E%E5%87%BD%E6%95%B0%E5%8F%AA%E6%89%A7%E8%A1%8C%E4%B8%80%E6%AC%A1.html",
    redirect: "/PAGE/深拷贝与函数只执行一次/"
  },
  {
    name: "v-0c2667c9",
    path: "/PAGE/String()，new String()以及单引号与双引号/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-0c2667c9").then(next)
    },
  },
  {
    path: "/PAGE/String()，new String()以及单引号与双引号/index.html",
    redirect: "/PAGE/String()，new String()以及单引号与双引号/"
  },
  {
    path: "/%E5%89%8D%E7%AB%AF/javascript/2020-07-18-String()%EF%BC%8Cnew-String()%E4%BB%A5%E5%8F%8A%E5%8D%95%E5%BC%95%E5%8F%B7%E4%B8%8E%E5%8F%8C%E5%BC%95%E5%8F%B7.html",
    redirect: "/PAGE/String()，new String()以及单引号与双引号/"
  },
  {
    name: "v-3d41f7ea",
    path: "/PAGE/Object.defineProperty之数据劫持/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-3d41f7ea").then(next)
    },
  },
  {
    path: "/PAGE/Object.defineProperty之数据劫持/index.html",
    redirect: "/PAGE/Object.defineProperty之数据劫持/"
  },
  {
    path: "/%E5%89%8D%E7%AB%AF/javascript/2020-07-21-Object.defineProperty%E4%B9%8B%E6%95%B0%E6%8D%AE%E5%8A%AB%E6%8C%81.html",
    redirect: "/PAGE/Object.defineProperty之数据劫持/"
  },
  {
    name: "v-7e8a06cb",
    path: "/PAGE/函数柯里化/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-7e8a06cb").then(next)
    },
  },
  {
    path: "/PAGE/函数柯里化/index.html",
    redirect: "/PAGE/函数柯里化/"
  },
  {
    path: "/%E5%89%8D%E7%AB%AF/javascript/2020-07-21-%E5%87%BD%E6%95%B0%E6%9F%AF%E9%87%8C%E5%8C%96.html",
    redirect: "/PAGE/函数柯里化/"
  },
  {
    name: "v-5e9fa9b6",
    path: "/PAGE/利用Proxy实现数据绑定/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-5e9fa9b6").then(next)
    },
  },
  {
    path: "/PAGE/利用Proxy实现数据绑定/index.html",
    redirect: "/PAGE/利用Proxy实现数据绑定/"
  },
  {
    path: "/%E5%89%8D%E7%AB%AF/javascript/2020-07-21-%E5%88%A9%E7%94%A8Proxy%E5%AE%9E%E7%8E%B0%E6%95%B0%E6%8D%AE%E7%BB%91%E5%AE%9A.html",
    redirect: "/PAGE/利用Proxy实现数据绑定/"
  },
  {
    name: "v-f86bae42",
    path: "/PAGE/自定义arr.slice方法/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-f86bae42").then(next)
    },
  },
  {
    path: "/PAGE/自定义arr.slice方法/index.html",
    redirect: "/PAGE/自定义arr.slice方法/"
  },
  {
    path: "/%E5%89%8D%E7%AB%AF/javascript/2020-07-21-%E8%87%AA%E5%AE%9A%E4%B9%89arr.slice%E6%96%B9%E6%B3%95.html",
    redirect: "/PAGE/自定义arr.slice方法/"
  },
  {
    name: "v-68985a74",
    path: "/PAGE/使用defineProperty实现Proxy/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-68985a74").then(next)
    },
  },
  {
    path: "/PAGE/使用defineProperty实现Proxy/index.html",
    redirect: "/PAGE/使用defineProperty实现Proxy/"
  },
  {
    path: "/%E5%89%8D%E7%AB%AF/javascript/2020-07-22-%E4%BD%BF%E7%94%A8defineProperty%E5%AE%9E%E7%8E%B0Proxy.html",
    redirect: "/PAGE/使用defineProperty实现Proxy/"
  },
  {
    name: "v-2ba563e9",
    path: "/PAGE/3种版本ajax的封装/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-2ba563e9").then(next)
    },
  },
  {
    path: "/PAGE/3种版本ajax的封装/index.html",
    redirect: "/PAGE/3种版本ajax的封装/"
  },
  {
    path: "/%E5%89%8D%E7%AB%AF/javascript/2020-08-01-3%E7%A7%8D%E7%89%88%E6%9C%AC%E7%9A%84ajax%E5%B0%81%E8%A3%85.html",
    redirect: "/PAGE/3种版本ajax的封装/"
  },
  {
    name: "v-730049a4",
    path: "/PAGE/Array.prototype.filter/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-730049a4").then(next)
    },
  },
  {
    path: "/PAGE/Array.prototype.filter/index.html",
    redirect: "/PAGE/Array.prototype.filter/"
  },
  {
    path: "/%E5%89%8D%E7%AB%AF/javascript/2020-08-01-Array.prototype.filter%E7%94%A8%E6%B3%95.html",
    redirect: "/PAGE/Array.prototype.filter/"
  },
  {
    name: "v-6f95e324",
    path: "/PAGE/Array.prototype.reduce/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-6f95e324").then(next)
    },
  },
  {
    path: "/PAGE/Array.prototype.reduce/index.html",
    redirect: "/PAGE/Array.prototype.reduce/"
  },
  {
    path: "/%E5%89%8D%E7%AB%AF/javascript/2020-08-01-Array.prototype.reduce%E7%94%A8%E6%B3%95.html",
    redirect: "/PAGE/Array.prototype.reduce/"
  },
  {
    name: "v-6a253be0",
    path: "/PAGE/生成器/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-6a253be0").then(next)
    },
  },
  {
    path: "/PAGE/生成器/index.html",
    redirect: "/PAGE/生成器/"
  },
  {
    path: "/%E5%89%8D%E7%AB%AF/javascript/2020-08-01-%E7%94%9F%E6%88%90%E5%99%A8.html",
    redirect: "/PAGE/生成器/"
  },
  {
    name: "v-3618bdb0",
    path: "/PAGE/js中的继承/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-3618bdb0").then(next)
    },
  },
  {
    path: "/PAGE/js中的继承/index.html",
    redirect: "/PAGE/js中的继承/"
  },
  {
    path: "/%E5%89%8D%E7%AB%AF/javascript/2020-08-02-js%E4%B8%AD%E7%9A%84%E7%BB%A7%E6%89%BF.html",
    redirect: "/PAGE/js中的继承/"
  },
  {
    name: "v-a2740946",
    path: "/PAGE/Promise简单实现/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-a2740946").then(next)
    },
  },
  {
    path: "/PAGE/Promise简单实现/index.html",
    redirect: "/PAGE/Promise简单实现/"
  },
  {
    path: "/%E5%89%8D%E7%AB%AF/javascript/2020-08-01-Promise%E7%AE%80%E5%8D%95%E5%AE%9E%E7%8E%B0.html",
    redirect: "/PAGE/Promise简单实现/"
  },
  {
    name: "v-b5e194d8",
    path: "/PAGE/JS实现异步编程的4种方法/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-b5e194d8").then(next)
    },
  },
  {
    path: "/PAGE/JS实现异步编程的4种方法/index.html",
    redirect: "/PAGE/JS实现异步编程的4种方法/"
  },
  {
    path: "/%E5%89%8D%E7%AB%AF/javascript/JS%E5%AE%9E%E7%8E%B0%E5%BC%82%E6%AD%A5%E7%BC%96%E7%A8%8B%E7%9A%844%E7%A7%8D%E6%96%B9%E6%B3%95.html",
    redirect: "/PAGE/JS实现异步编程的4种方法/"
  },
  {
    name: "v-13a5b318",
    path: "/PAGE/JavaScript创建对象的方法/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-13a5b318").then(next)
    },
  },
  {
    path: "/PAGE/JavaScript创建对象的方法/index.html",
    redirect: "/PAGE/JavaScript创建对象的方法/"
  },
  {
    path: "/%E5%89%8D%E7%AB%AF/javascript/JavaScript%E5%88%9B%E5%BB%BA%E5%AF%B9%E8%B1%A1%E7%9A%84%E6%96%B9%E6%B3%95.html",
    redirect: "/PAGE/JavaScript创建对象的方法/"
  },
  {
    name: "v-2a3f2f31",
    path: "/PAGE/数字签名是什么？/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-2a3f2f31").then(next)
    },
  },
  {
    path: "/PAGE/数字签名是什么？/index.html",
    redirect: "/PAGE/数字签名是什么？/"
  },
  {
    path: "/%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%BD%91%E7%BB%9C/2020-07-13-%E6%95%B0%E5%AD%97%E7%AD%BE%E5%90%8D%E6%98%AF%E4%BB%80%E4%B9%88%EF%BC%9F.html",
    redirect: "/PAGE/数字签名是什么？/"
  },
  {
    name: "v-4d3f09f0",
    path: "/PAGE/MyProfile/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-4d3f09f0").then(next)
    },
  },
  {
    path: "/PAGE/MyProfile/index.html",
    redirect: "/PAGE/MyProfile/"
  },
  {
    path: "/%E7%AE%80%E4%BB%8B/MyProfile.html",
    redirect: "/PAGE/MyProfile/"
  },
  {
    name: "v-84fa1624",
    path: "/PAGE/为什么TCP是可靠的？/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-84fa1624").then(next)
    },
  },
  {
    path: "/PAGE/为什么TCP是可靠的？/index.html",
    redirect: "/PAGE/为什么TCP是可靠的？/"
  },
  {
    path: "/%E8%AE%A1%E7%AE%97%E6%9C%BA%E7%BD%91%E7%BB%9C/2020-07-27-%E4%B8%BA%E4%BB%80%E4%B9%88TCP%E6%98%AF%E5%8F%AF%E9%9D%A0%E7%9A%84%EF%BC%9F.html",
    redirect: "/PAGE/为什么TCP是可靠的？/"
  },
  {
    name: "v-52aa54a1",
    path: "/PAGE/前端工程化的理解/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-52aa54a1").then(next)
    },
  },
  {
    path: "/PAGE/前端工程化的理解/index.html",
    redirect: "/PAGE/前端工程化的理解/"
  },
  {
    path: "/%E5%89%8D%E7%AB%AF/javascript/%E5%89%8D%E7%AB%AF%E5%B7%A5%E7%A8%8B%E5%8C%96%E7%9A%84%E7%90%86%E8%A7%A3.html",
    redirect: "/PAGE/前端工程化的理解/"
  },
  {
    name: "v-06a5cf42",
    path: "/%E5%89%8D%E7%AB%AF/javascript/%E5%89%8D%E7%AB%AF%E4%BD%93%E7%B3%BBinterview%E6%80%BB%E7%BB%93.html",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-06a5cf42").then(next)
    },
  },
  {
    path: "/前端/javascript/前端体系interview总结.html",
    redirect: "/%E5%89%8D%E7%AB%AF/javascript/%E5%89%8D%E7%AB%AF%E4%BD%93%E7%B3%BBinterview%E6%80%BB%E7%BB%93.html"
  },
  {
    name: "v-44d77f16",
    path: "/PAGE/前端体系interview总结/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-44d77f16").then(next)
    },
  },
  {
    path: "/PAGE/前端体系interview总结/index.html",
    redirect: "/PAGE/前端体系interview总结/"
  },
  {
    path: "/%E5%89%8D%E7%AB%AF%E4%BD%93%E7%B3%BBinterview%E6%80%BB%E7%BB%93/%E5%89%8D%E7%AB%AF%E4%BD%93%E7%B3%BBinterview%E6%80%BB%E7%BB%93.html",
    redirect: "/PAGE/前端体系interview总结/"
  },
  {
    name: "v-77acd46e",
    path: "/PAGE/深度学习介绍/",
    component: GlobalLayout,
    beforeEnter: (to, from, next) => {
      ensureAsyncComponentsLoaded("Layout", "v-77acd46e").then(next)
    },
  },
  {
    path: "/PAGE/深度学习介绍/index.html",
    redirect: "/PAGE/深度学习介绍/"
  },
  {
    path: "/%E6%B7%B1%E5%BA%A6%E5%AD%A6%E4%B9%A0/01.%E6%B7%B1%E5%BA%A6%E5%AD%A6%E4%B9%A0%E7%9A%84%E4%BB%8B%E7%BB%8D.html",
    redirect: "/PAGE/深度学习介绍/"
  },
  {
    path: '*',
    component: GlobalLayout
  }
]
